src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -53,18 +53,20 @@ public QuoraProfileScraper() {
         super();
```

Comment:
I have made no changes in QuoraProfileScraper, these changes are to prevent any error, if it occurs due to configurations in GithubProfileScraper. Before GithubProfileScraper, BaseScraper was only used in QuoraProfileScraper and it is new, so to be make it more generic some changes are necessary.